lua/neorg/modules/core/qol/todo_items/module.lua

```diff
@@ -96,7 +99,7 @@ module.config.public = {
     -- - Test Item
     -- ```
     -- With this option set to true, performing an operation (like pressing `<C-space>`
-    -- or `gtu` or what have you) will convert the non-todo item into one:
+    -- or what have you) will convert the non-todo item into one:
```

Comment:
1. I'm not even sure if this is correct English, it sounds very weird to me (I'm not a native speaker though)
2. I'd just write "or another keybinding to change todo states"

Reply:
This change is just fine. It's valid english haha

Reply:
😳 ok then
but I still think you could add what I said

Reply:
I believe it's correct. But to make it clearer to even non native English speaker
how about this?

```suggestion
-- or what have you) will convert the non-todo item into one:
++ or your custom keybind) will convert the non-todo item into one:
```

Reply:
well like this you could think that it's just about the cycle thing or however the `<c-space>` is called
but it's also for `<localleader>td|p|u|c` etc

Reply:
I will just stick with "what have you"